en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md

```diff
@@ -34,127 +34,110 @@ Follow these steps to set up the Integration Project and import AmazonS3 connect
 
 {!reference/connectors/importing-connector-to-integration-studio.md!}
 
-1. Right click on the created Integration Project and select, -> **New** -> **Rest API** to create the REST API.
-   <img src="{{base_path}}/assets/img/integrate/connectors/adding-an-api.jpg" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
+## Creating the Integration Logic
 
-2. Specify the API name as `S3ConnectorTestAPI` and API context as `/s3connector`. You can go to the source view of the XML configuration file of the API and copy the following configuration.
+1. Specify the API name as `S3ConnectorTestAPI` and API context as `/s3connector`.
+
+2. First we will create the `/createbucket` resource. This API resource will retrieve the bucket name from the incoming HTTP PUT request and create a bucket in Amazon S3.
+Right click on the API Resource and go to **Properties** view. We use a URL template called `/createbucket` as we have three API resources inside a single API. The method will be PUT.
```

Comment:
I don't think the numbering will work unless we tab this.

Reply:
It's a single step, I have mistakenly added a new line. Fixed with a27bfd5